client/jetpack-cloud/sections/agency-dashboard/downtime-monitoring/toggle-activate-monitoring/index.tsx

```diff
@@ -54,22 +62,63 @@ export default function ToggleActivateMonitoring( { site, status, settings }: Pr
 					},
 					comment: '%(minutes) is the no of minutes, e.g. "5m"',
 			  } );
+
+		const currentSchedule = hours
+			? translate( '%(hours)d hour', '%(hours)d hours', {
+					count: hours,
+					args: {
+						hours,
+					},
+					comment: '%(hours) is the no of hours, e.g. "1 hour"',
+			  } )
+			: translate( '%(minutes)d minute', '%(minutes)d minutes', {
+					count: minutes,
+					args: {
+						minutes,
+					},
+					comment: '%(minutes) is the no of minutes, e.g. "5 minutes"',
+			  } );
 		return (
 			<div className="toggle-activate-monitoring__duration">
-				<img src={ clockIcon } alt="Monitoring duration" />
-				<span>{ currentDurationText }</span>
+				<Button
+					borderless
+					compact
+					onClick={ handleToggleNotificationSettings }
+					aria-label={
+						translate(
+							'The current notification schedule is set to %(currentSchedule)s. Click here to update the settings',
+							{
+								args: { currentSchedule },
+								comment:
+									'%(currentSchedule) is the current notification duration set, e.g. "1 hour" or "5 minutes"',
+							}
+						) as string
```

Comment:
Out of curiosity: why are we casting the returned value of `translate` to string here?

Reply:
`<Button>` component expects the aria-label to be a string, whereas the translate is not considered a string but a ReactChild.